dev/doc/xml-protocol.md

```diff
@@ -753,3 +754,41 @@ Ex: `status = "Idle"` or `status = "proof: myLemmaName"` or `status = "Dead"`
 </feedback>
 ```
 
+## <a name="highlighting">Highlighting</a>
```

Comment:
Can you make sure that your title here is consistent with the one you put in the TOC?

Also, note that despite what has been done in this file, you don't need to go to such length to create section titles that can be linked to. See for instance, `CONTRIBUTING.md`.

Reply:
Will do.

As a general principle, I try to follow the style that's already used in a document (or code) rather than following a preferred format/model and making the document/code less consistent.  With exceptions, of course.  Perhaps many exceptions.